src/engine/include/models/mesh_material.hpp

```diff
@@ -65,13 +66,18 @@ class MeshMaterial
      * Returns bounding box (AABB).
      * Total length: 8
      */
-    Vector3 *getBoundingBox() { return boundingBox; };
+    Vector3 *getBoundingBoxVertices() { return boundingBoxObj->getVertices(); };
 
     /** 
      * Returns bounding box (AABB) vertex.
      * Total length: 8
      */
-    Vector3 &getBoundingBoxVertex(const u8 &i) { return boundingBox[i]; };
+    Vector3 &getBoundingBoxVertex(const u8 &i) { return boundingBoxObj->getVertex(i); };
+
+    /**
+     * Returns bounding box (AABB) object pointer.
```

Comment:
You can use @returns when you will use it, doxygen will automatically generate documentation (in future)